ProcessMaker/process-templates
# FOUR-12383 Update Template Elements

## Description:

This pull request addresses the Ticket for Updating new template files.
## Changes Made:

Revert `uuid` in some of the updated templates to prevent duplication
Add or update `version `to some of the templates in the manifest files
Update missing `key` to "default_templates" in some of the manifest files
Update `user_id` to null in some of the manifest files

## How to test

1. Delete all templates from the database.
2. Update your `.env` to fetch the develop branch of the templates repo first.
```
DEFAULT_TEMPLATE_REPO=process-templates
DEFAULT_TEMPLATE_BRANCH=develop
DEFAULT_TEMPLATE_CATEGORIES=all
```
sync the templates `php artisan processmaker:sync-default-templates`
3. Check if the current templates are synced.
4. Update your `.env` to fetch the new templates from this branch `DEFAULT_TEMPLATE_BRANCH=FOUR-12383`
```
DEFAULT_TEMPLATE_REPO=process-templates
DEFAULT_TEMPLATE_BRANCH=FOUR-12383
DEFAULT_TEMPLATE_CATEGORIES=all
```
sync the templates `php artisan processmaker:sync-default-templates`
5. Please ensure that the new templates have been added and no duplicates have been included. 
-- eg: `Expense Approval 2`

Ticket [FOUR-12383](https://processmaker.atlassian.net/browse/FOUR-12383)


[FOUR-12383]: https://processmaker.atlassian.net/browse/FOUR-12383?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
